.github/workflows/all_plugins.yaml

```diff
@@ -0,0 +1,62 @@
+name: all_plugins
+
+on:
```

Comment:
Add a step for testing publishing.  In my other package we found this check quick and very helpful in case we messed up with the project structure.  Also I think we should consider building the apps (can be in a pr later of course)  since we have quite some native code.

- [ ] Create job which runs `pub publish --dry-run` for packages

Reply:
Thanks Remon, will do this in a separate PR and created this issue like we discussed offline
https://github.com/flutter-stripe/flutter_stripe/issues/52

Reply:
Let's add it once melos is merged